ledger/acctupdates.go

```diff
@@ -246,66 +255,36 @@ func (au *accountUpdates) waitAccountsWriting() {
 	au.accountsWriting.Wait()
```

Comment:
The error on line 244 is currently ignored, could you please add an `if err != nil` check there?

Reply:
yes! good find.